### PR TITLE
chore: use next image to avoid layout shift

### PR DESCRIPTION
**Pull request by Chef-Yogi**
close PAN-291

### CODE REVIEW

File: src/views/Home/components/Hero.tsx [hunk 1]
```diff
@@ -125,12 +132,15 @@ const Hero = () => {
...
-              <img src={`${imagePath}${imageSrc}.png`} alt={t('Lunar bunny')} />
-            </picture>
+          <BunnyWrapper ref={bunnyWrapperRef}>
+            <Image
+              src={bunnyImage}
+              onLoad={() => {
+                if (!bunnyWrapperRef.current) return
+                bunnyWrapperRef.current.classList.add('appear')
+              }}
+              placeholder="blur"
```
Comment (chef-jojo) [inline src/views/Home/components/Hero.tsx:142]: add `priority` to not lazy loading this

File: src/views/Home/components/Hero.tsx [hunk 2]
```diff
@@ -51,6 +54,12 @@ const InnerWrapper = styled.div`
 const BunnyWrapper = styled.div`
   width: 100%;
   animation: ${flyingAnim} 3.5s ease-in-out infinite;
+  transition: opacity 0.35s ease-in;
+  will-change: opacity, transform;
+  opacity: 0;
+  &.appear {
+    opacity: 1;
```
Comment (chef-jojo) [inline src/views/Home/components/Hero.tsx:61]: is it better to not animation this?

**Comment by chef-jojo** [comment]
can also remove bunny.webp in public folder

File: src/views/Home/components/Hero.tsx (hunk 2 above)
Comment (Chef-Yogi) [reply src/views/Home/components/Hero.tsx]: removed!

File: src/views/Home/components/Hero.tsx (hunk 1 above)
Comment (Chef-Yogi) [reply src/views/Home/components/Hero.tsx]: Done

**Review by chef-jojo — APPROVED** [review]

**Pull request merged**